x-pack/plugins/security_solution/public/timelines/components/timeline/query_bar/index.tsx

```diff
@@ -47,6 +48,42 @@ export interface QueryBarTimelineComponentProps {
   updateReduxTime: DispatchUpdateReduxTime;
 }
 
+const SearchBarContainer = styled.div`
+  /*
+  *
+  * hide search bar default filters as they are distrubing the layout  as shown below
+  *
+  * Filters are displayed with QueryBar so below is how is the layout with default filters.
+  *
+  *
+  *                    --------------------------------
+  *   -----------------|                              |------------
+  *   | DataViewPicker |        QueryBar              | Date      |
+  *   -------------------------------------------------------------
+  *                    |      Filters                 |
+  *                    --------------------------------
+  *
+  * The tree under this component makes sure that default filters are not rendered and we can saperately display
```

Comment:
saperately => separately

Reply:
This [spell checker extension](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) for VSCode works awesome for me. Other IDEs should have similar things.

Reply:
yes, I have that as well but some gets left behind since pre-commit check doesn't include that check. Will try to see if include that locally.

Reply:
fixed here: 05b43e24fd3453df763165de4bf2822a10193421